README.md

```diff
@@ -24,6 +24,28 @@ Also note that the version no longer follows SonarQube's versions. The `sonar-pl
 
 See the [releases](https://github.com/SonarSource/sonar-plugin-api/releases) for changelogs.
 
+## Compatibility
+
+Breaking changes to the API happen when the major version changes. As a result, a plugin compiled with a certain version of the API should work with  platforms implementing an API of the same major version. 
```

Comment:
I'm worried this could cause confusion, as it contradicts the sentence on line 30 to an extent. I can compile a plugin with a given major that will not run on a platform implementing an API of the same major version because I used a brand new API that is not available on the platform due to an older minor version. Perhaps it is worth explicitly stating that *platforms* don't break compatibility as long as they don't upgrade to a new major version? Plugins can, in fact, be non-backwards compatible even if they don't upgrade a major version.

Reply:
Ok, took me a bit to understand but I got your point :)

Reply:
I'm not sure about "platforms breaking compatibility", though. Platforms shouldn't matter for plugins.
I'll suggest a different sentence and LMK what you think

Reply:
Sounds good! :+1: